p2p/src/simulated/network.rs

```diff
@@ -36,43 +41,34 @@ type Task = (
     oneshot::Sender<Vec<PublicKey>>,
 );
 
+/// Configuration for the simulated network.
+pub struct Config {
+    /// Registry for prometheus metrics.
+    pub registry: Arc<Mutex<Registry>>,
+
+    /// Maximum size of a message that can be sent over the network.
+    pub max_size: usize,
+}
+
 /// Implementation of a simulated network.
-pub struct Network<E: Spawner + Rng + Clock> {
+pub struct Network<E: RNetwork<Listener, Sink, Stream> + Spawner + Rng + Clock> {
```

Comment:
nit: instead of doing something like `RNetwork`, I think it is clearer to use the import path to name it `commonware_runtime::Network`/use a more descriptive name like `RuntimeNetwork`

Reply:
This patten was followed from `src/authenticated` :)

Should I change it in both places or just leave it for now?